### PR TITLE
Added `/popup` command to open currently focused split or supplied channel in a new window

### LINKED ISSUE #3517: Add a `/popup` command
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Add a `/popup [channel]` command
Works similar to the `/popout` command - basically allowing a user to popup a chat without having to open the split. Using the command without the channel name would open the current split in a popup.

### Screenshots

_No response_

### OS and Chatterino Version

Chatterino Nightly 7.3.4 (commit 8e7fe9910) on Windows 10

### PR DESCRIPTION
**Pull request by qooq69**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
Closes #3517

### CODE REVIEW

File: src/controllers/commands/CommandController.cpp [hunk 1]
```diff
@@ -697,6 +697,33 @@ void CommandController::initialize(Settings &, Paths &paths)
...
 
+    this->registerCommand(
+        "/popup", [](const QStringList &words, ChannelPtr channel) {
+            QString target(words.size() < 2 ? channel->getName() : words[1]);
+
+            if (words.size() < 2 &&
+                (!channel->isTwitchChannel() || channel->isEmpty()))
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Usage: /popup <channel>. You can also use the command "
```
Comment (ALazyMeme) [inline src/controllers/commands/CommandController.cpp:708]:
```suggestion
                    "Usage: /popup [channel]. You can also use the command "
```

Comment (ALazyMeme) [reply src/controllers/commands/CommandController.cpp:708]: Refer here: https://github.com/Chatterino/chatterino2/blob/1d272c399b805f75284bbf6db0511243cd5ccc88/CONTRIBUTING.md#conventions

Comment (Felanbird) [inline src/controllers/commands/CommandController.cpp:708]:
```suggestion
                    "Usage: /popup [channel]. You can also use the command "
```
`[]` optional vs `<>` required - https://github.com/Chatterino/chatterino2/pull/3530#issuecomment-1019238728 for extra reference

**Review by Felanbird — APPROVED** [review]
Otherwise Functionality works as intended. 👍

File: src/controllers/commands/CommandController.cpp (hunk 1 above)
Comment (Felanbird) [reply src/controllers/commands/CommandController.cpp:708]: Actually, with the extra explanation, this one might be an actual technicality.

**Comment by pajlada** [comment]
This shares some (but not all) code with [Split::popup](https://github.com/Chatterino/chatterino2/blob/51ece94f588a60c578504309f9eeea22a831f0fa/src/widgets/splits/Split.cpp#L876-L890) - any way we could use the Split::popup() code where applicable? like if the user has not specified a channel name parameter.

**Comment by pajlada** [comment]
I changed it @qooq69 - take a look if this makes sense! Code duplication is back, but it's the only way I could see not having a random Split created and forgotten about

**Comment by qooq69** [comment]
I have made a minor change of using the createwindow() optional value to show the new window. Otherwise looks good to me! :+1:

**Review by pajlada — APPROVED** [review]

**Pull request merged**